index.js

```diff
@@ -39,9 +39,12 @@ module.exports = function(content) {
 	}
 	var root = config.root;
 	var links = attrParse(content, function(tag, attr) {
-		var item = tag + ":" + attr;
 		var res = attributes.find(function(a) {
-		  return item.indexOf(a) >= 0;
+			if (a.startsWith(':')) {
```

Comment:
@michael-ciniawsky ⚠️  FYI `startsWith` method has been introduced in `ES6` so... not sure if it's ok or I should replace it with the corresponding `ES5` version ...